packages/render-fidelity-tools/src/components/rendering-scenario.ts

```diff
@@ -80,22 +95,11 @@ export class RenderingScenario extends LitElement {
                     style="width:${width}px" src="${basePath}/${golden.file}">
                   <div class = "metrics">
                     ${
-                    index == 0 || this.analysis == null ?
-                        html` <span> --- </span>` :
-                        html` <span>${
-                            this.toDecibels(
-                                this.analysis.analysisResults[index - 1][0]
-                                    .rmsDistanceRatio)} dB </span>`}
-                    <div class="tooltip">
-                      <span class="question-icon"> </span>
-                      <span class="tooltiptext">
-                        Root mean square(RMS) color distance between ${
-                    golden.name} 
-                        and current version of model-viewer on rendering ${
-                    this.name} in decibels.
-                        The decibel is given by: 10 * log(RMS). More negative means a closer match.
-                      </span>
-                    </div>
+                               index == 0 || this.analysis == null ?
+                                   html`<span>---</span>` :
+                                   this.metricTemplate(
+                                       this.analysis.analysisResults[index - 1],
+                                       golden.name)}
```

Comment:
@prideout  I saw you use Mustache template to make html very clean(I learn a lot from your code). But i found mustache doesn't support typescript, and since this web component is based on lit-element , it's not easy to use query selector (actually i didn't find out how to use it) here. So the best way i can think of to refactor this is just use a template function here and that's also the way I found in lit-element's document. If you have better way to do it im more than happy to learn

Reply:
No no, lit-element looks great, sorry for the distraction.  :)